### ISSUE TITLE
Scan tests fail at runtime with MSVC

### ISSUE THREAD

**Issue by davidbeckingsale** [LLNL/RAJA#168]
First seen in #145

**Comment by willkill07** [comment]
Is there a way to easily toy around with AppVeyor using a config file? It would be nice to be able to easily test to see if we can re-enable scans now that we are on the latest MSVC

**Comment by davidbeckingsale** [comment]
AFAIK the AppVeyor setup is managed by @trws - if you remove the check that guards the scans for MSVC and push a PR, AppVeyor will try and build it.

**Comment by trws** [comment]
I can give you access @willkill07 but it's not quite as convenient as a file you can edit and commit.  We can add a script to the repository, or a cmake target, that would get us there, but it's a script in a web interface generally.

**Comment by trws** [comment]
I happily take that back!  Apparently they have that now, this is our current configuration exported as yaml:

```yaml
version: 1.0.{build}
skip_branch_with_pr: true
image: Visual Studio 2017
build_script:
- cmd: >-
    mkdir build

    cd build

    cmake ../

    cmake --build . --config Release
test_script:
- cmd: ctest -VV -C Release
```

Plop that into an `appveyor.yaml` file in the root directory and we can tweak it the same way travis is done, and by-branch as well (supposedly, I have not tested this).

**Comment by davidbeckingsale** [comment]
I'll add a PR that just adds the file.

**Comment by trws** [comment]
Fixed by #306 IIRC.
